index.html

```diff
@@ -109,9 +109,9 @@ <h3>
     <!-- scheme://prefix.domain:port/path/filename
 -->
     <p>
-      Wikipedia: https://en.wikipedia.org/wiki/Duck <br>
-      Facts: https://www.four-paws.org/campaigns-topics/topics/farm-animals/10-facts-about-ducks <br>
-      Some random ice-hockey team: https://www.nhl.com/ducks <br> <br>
+      <a href="https://en.wikipedia.org/wiki/Duck">The Wikipedia of Ducks</a> <br>
+      <a href="https://www.four-paws.org/campaigns-topics/topics/farm-animals/10-facts-about-ducks">Some more facts about Ducks</a> <br>
+      <a href="https://www.nhl.com/ducks">A random hockey team called Ducks</a> <br> <br>
```

Comment:
ist das zweite `<br>` nicht redundant?

Reply:
das zweite ist für ein schöneres styling

Reply:
Ah okay. Wir sollten uns in Zukunft mal Styling Möglichkeiten mit css ansehen. Dann können wir deinen Codestyle gleich mit verschönern ;)